resolvers/webpack/package.json

```diff
@@ -27,7 +27,7 @@
   "bugs": {
     "url": "https://github.com/benmosher/eslint-plugin-import/issues"
   },
-  "homepage": "https://github.com/benmosher/eslint-plugin-import#readme",
+  "homepage": "https://github.com/benmosher/eslint-plugin-import/tree/master/resolvers/webpack",
```

Comment:
hardcoding "master" here is dangerous; I think maybe https://github.com/benmosher/eslint-plugin-import#resolvers is a better place to link to

Reply:
I was just following the pattern in the babel lerna repo. I very much doubt git's going to change the  default branch from master any time soon.

Reply:
github wouldn't, but any maintainer might - that's not the danger i'm concerned with. I don't want to link to a directory that might disappear in a future refactoring.

Reply:
I get what you're saying @ljharb, but this looks reasonable enough, especially if this is a well-known lerna pattern.